services/community-site/src/pages/RunNode/RunNode.tsx

```diff
@@ -195,6 +199,13 @@ const RunNode = () => {
       return;
     }
 
+    const testnetNodesRequest = await delegationApi.get(`/nodes?type=testnet`, true);
```

Comment:
This is a bit confusing since you don't reuse it later on, wouldn't a simple `useEffect((), [])` be more vocal about the use-case? Tho if you wanna leave the implementation like this these events can be also fired when the user toggles between the views, which would be a nice feature. 🌮

Reply:
👍

Reply:
Updated this to be called in another useEffect.